### DISCUSSION TITLE
Bug in  Evaluation

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#69] [BaiduKDDCup2022] [answered]
We found that if we provide nan for prediction, the mse and mae will be zero. Therefore, if we set all prediction as nan, the online score will be 0. I think it should be fixed as soon as possible.

**Comment by ramber1836** [comment]
Indeed, this is a bug, we will fix it ASAP, and update the online
evaluation accordingly.
Please note the announcement on the official website.

**Reply by light101010** [reply]
After the online evaluation is updated, it is suggested to empty the Leaderboard and use the new evaluation for the results of subsequent rankings.

**Comment by ramber1836** [comment]
The false high results will be updated.

**Comment by zhoujingbo** [comment, marked as answer]
The online tests have been updated to avoid the false high scores, please refer to the GitHub(https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation) updates for more details.
